tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -146,39 +147,54 @@ func (d *deployer) createCluster(zones []string, adminAccess string, yes bool) e
 		}
 	}
 	if !foundCPCount {
-		args = appendIfUnset(args, "--master-count", fmt.Sprintf("%d", d.ControlPlaneCount))
+		args = appendIfUnset(args, "--control-plane-count", fmt.Sprintf("%d", d.ControlPlaneCount))
```

Comment:
We need to use `master` instead of `control-plane` for backwards compatibility. Please revert.

Reply:
Fixed

Reply:
Thanks :)

Reply:
Please do that everywhere, not just this specific flag.

Reply:
Revert the changes to all the flags

Reply:
There is still a mention of `--control-plane-size`.
Please remove the new "change this deprecated flag after the tester starts using kops.k8s.io/v1alpha3 APIs" comments. It is not correct.